frontend/common/src/components/Auth/AuthorizationContainer.tsx

```diff
@@ -2,28 +2,34 @@ import React from "react";
 import { Role } from "../../api/generated";
 
 export type PossibleUserRoles = (Role | null | undefined)[] | null | undefined;
+export type PossibleEmail = string | null;
```

Comment:
```suggestion
import { Maybe, Role } from "../../api/generated";

export type PossibleUserRoles = (Role | null | undefined)[] | null | undefined;
export type MaybeEmail = Maybe<string>;
```

Thoughts on using `Maybe<T>`? It's [pretty well-established](https://en.wikipedia.org/wiki/Monad_(functional_programming)#An_example:_Maybe) beyond our project (though we don't go full-monad :) )

This keeps the maybe/possible types consistent across the project

Reply:
Makes sense to me, I was just following the pattern established in the file with `PossibleRoles`. Committed suggestion 👍